ams/ams-server/src/main/java/com/netease/arctic/ams/server/service/impl/OptimizeQueueService.java

```diff
@@ -74,7 +79,7 @@
 public class OptimizeQueueService extends IJDBCService {
   private static final Logger LOG = LoggerFactory.getLogger(OptimizeQueueService.class);
 
-  private final Map<Integer, OptimizeQueueWrapper> optimizeQueues = new HashMap<>();
+  private static final Map<Integer, OptimizeQueueWrapper> optimizeQueues = new HashMap<>();
```

Comment:
why optimizeQueues set to static?

Reply:
Because getQueueId and getOptimizeQueue set static.
I will fix it.